backend/WebApplication1/Infrastructure/Utils/ImageFromFileToBytes.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.IO;
+
+namespace WebApplication1.Infrastructure.Utils
+{
+    public class ImageFromFileToBytes
```

Comment:
static class

Reply:
also, don't make it so specialized, call it smth like ImageToBytes, so later on you can extend it and inject the way of getting image (not only from files)

Reply:
or ImageConverter would also work